src/components/PlanListPage.jsx

```
@@ -0,0 +1,157 @@
+      {/* 회색 로그인 안내 영역 */}
+      <section className="login-banner">
+        <span>로그인하고 현재 가입 조건으로 이용하세요.</span>
+        <button>로그인하기</button>
+      </section>
+
+      {/* 플랜 종류 네비게이션 */}
+      <ul className="plan-type-nav">
+        {PLAN_TYPES.map(pt => (
+          <li
+            key={pt.value}
+            className={pt.value === planType ? 'active' : ''}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)